pjproject/2.12.1/MD5SUM.TXT

```diff
@@ -0,0 +1,3 @@
+7f80ba8e1540853f959be6be7912a150  pjproject-2.12.1.tar.bz2
+339fcf1373276184fbdb821f75a1efbf  pjproject-2.12.1.tar.gz
+e086683e3441be6e609fc4ce8cf2757f  pjproject-2.12.1.zip
```

Comment:
The checksums for the tar.gz and zip files don't match what I get when I run md5sum...
```
a0250481bb082d553ae671bee7519d1b  pjproject-2.12.1.tar.gz
94823f8556403b72e47486089c1cad96  pjproject-2.12.1.zip
```
The bz2 and sha256 checksums are OK though.